src/containers/shared/css/global.scss

```diff
@@ -3,17 +3,17 @@
 @import './variables';
 @import './form';
 
-/* prettier-ignore */
+/**
+ * `currency_symbols` defines currencies fonts are missing, currently `u+e900` (XRP) and `u+e901` (BTC).
+ * By using unicode-range we specify `currency_symbols` at the top level and any other characters will fall through to
+ * the next font in the stack, in this case Work Sans.
```

Comment:
super nit - i think this comment can be more descriptive and clear. the "falling through" that you mention is not defined here by the font-face itself or the use of `unicode-range`. perhaps something like

```text
`current_symbols` is used for currency codes that are missing in other font definitions, currently .....

By using unicode-range we specify which character points this font-family can be used for. When used in a style (and assuming this font family is first/takes precedent), only these characters will use this font-face. Anything else will fall through to the subsequent font-family.
```